code/core/src/preview-api/modules/store/args.ts

```
@@ -54,6 +54,17 @@ const map = (arg: unknown, argType: InputType): any => {
         const mapped = map(val, { type: type.value[key] });
         return mapped === INCOMPATIBLE ? acc : Object.assign(acc, { [key]: mapped });
       }, {} as Args);
+    case 'other':
+      const isPrimitiveArg =
+        typeof arg === 'string' || typeof arg === 'number' || typeof arg === 'boolean';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)